az-storage-account: add page (#8737)
tldr-pages/tldr: pages/common/az-storage-account.md

# az storage account

> Manage storage accounts in Azure.
> Part of `azure-cli`.
> More information: <https://learn.microsoft.com/en-us/cli/azure/storage/account>.

> - Create a account in a storage account:

Reviewer: Wait, based on the doc this command is for create an storage account itself :thinking: 
```suggestion
- Create an storage account:
```

`az storage account create --name {{storage_account_name}} --resource-group {{azure_resource_group}} --location {{azure_location}} --sku {{storage_account_sku}}`

> - Generate a shared access signature for the account:

Reviewer: ```suggestion
- Generate a shared access signature for a specific storage account:
```

`az storage account generate-sas --account-name {{storage_account_name}} --name {{account_name}} --permissions {{sas_permissions}} --expiry {{expiry_date}} --services {{storage_services}} --resource-types {{resource_types}}`

- List storage accounts:

`az storage account list --resource-group {{azure_resource_group}}`

> - Delete a storage account:

Reviewer: ```suggestion
- Delete a specific storage account:
```
